### PR TITLE
Admin/scripts

### PR DESCRIPTION
**Pull request by jim-parry**
Version 2 of the release build scripts.
Tested locally, and all appears well (famous last words).
I won't merge & run this until you have a chance to comment on it.
First live run will be manual for "release-deploy" :-/
@lonnieezell @natanfelles What do you think?

### CODE REVIEW

File: admin/release-revert [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Revert local repos to pre-release state
+echo -e "${BOLD}${COLOR}CodeIgniter4 release revert${NORMAL}"
+echo '---------------------------'
+
+if [ $# -lt 2 ]; then
+    echo "You forgot the magic word"
+    exit 1
+fi
+if [ $1 != 'please' ]; then
```
Comment (lonnieezell) [inline admin/release-revert]: You have to say please to revert? What a polite script :)

Comment (jim-parry) [reply admin/release-revert]: That is only to prevent inadvertent reverting. The other scripts need a version and/or qualifier.

Comment (natanfelles) [reply admin/release-revert]: First time I read, I ask: "what magic word?" haha amazing

**Comment by natanfelles** [comment]
Seems pretty good. *Please*, merge it! :)

**Comment by jim-parry** [comment]
@lonnieezell Should we continue to bundle the docs with the "framework" redistributable? They are available for separate download, which could be made composer-installable (it isn't at the moment). The released docs will also be available through github.io (and won't need hosting on our website).

Download sizes:
- framework with docs - 3.2MB gz or 3,6MB zip
- framework without docs - 455KB gz or 728KB zip
- docs - 2.7MB gz or 2.9MB zip
- appstarter - 44KB gz or 72KB zip

**Comment by lonnieezell** [comment]
I would just include the framework personally.

**Comment by jim-parry** [comment]
@lonnieezell It looks like travis-ci is not happy with your LiveConnectTest changes :(
1) CodeIgniter\Database\Live\ConnectTest::testConnectWithMultipleCustomGroups
ErrorException: Trying to get property of non-object
/home/travis/build/codeigniter4/CodeIgniter4/tests/system/Database/Live/ConnectTest.php:20

**Comment by lonnieezell** [comment]
That was corrected this morning, i believe. Re-run your tests at Travis for this and it should be good.

**Pull request merged**